DataFormats/ForwardDetId/src/MTDChannelIdentifier.cc

```diff
@@ -0,0 +1,33 @@
+// Modify the pixel packing to make 100micron pixels possible. d.k. 2/02
+//
+#include "DataFormats/ForwardDetId/interface/MTDChannelIdentifier.h"
+
+
+/*
+// Extract from CMSIM manual (version Thu Jul 31 16:38:50 MET DST 1997)
```

Comment:
is this comment from 21 years ago really needed?

Reply:
21-year old and referring to something else. I removed it.